### PR TITLE
Add Vietnamese language

### PR DESCRIPTION
**Pull request by NattapolChan**
## ✅ New Language Checklist
- [x] There is ❌ no assigned issue or pull request for this new language.
- [x] ➕ Added a new JSON object in the `server/data/languages.json` file.
- [x] Added the **voice** 🎤 attribute supported by ResponsiveVoice API.
- [x] Added the **flag** 🏳 attribute in the format of the twemoji-awesome CSS syntax.
- [ ] Added the **romanization** attribute for appropriate languages.
- [x] ➕ Created the `{language.json}` file inside the `server/data/words` folder containing the list of the 3000 words.
- [x] Removed duplicates and noted 🗒 them in the Pull Request.
- [x] Increment `currentLanguagesCount` in `client/src/store/index.js`
- [x] Add new language in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 - 🇸🇪 Swedish
 - 🇮🇳 Tamil
 - ᴛʜ Thai
+- ᴠ🇮 Vietnamese
```
Comment (kvharish) [inline README.md:53]: @NattapolChan  Could you please fix the flag here?

Comment (kvharish) [reply README.md]: I can see you have updated the pull request but the flag seems to have an issue still. Could you check it?

File: README.md [hunk 2]
```diff
@@ -49,8 +49,8 @@
 - 🇪🇸 Spanish
 - 🇸🇪 Swedish
 - 🇮🇳 Tamil
-- ᴛʜ Thai
-- ᴠ🇳 Vietnamese
+- 🇹🇭 Thai
```
Comment (kvharish) [inline README.md:52]: There is already an open pull request for fixing the Thailand flag. If your pull request gets merged before I will decline mine.

File: server/data/words/vietnamese.json [hunk 3]
```diff
@@ -0,0 +1,12002 @@
...
+    },
+    {
+      "word": "bản thân bạn",
+      "translation": "youth"
+    },
+    {
+      "word": "thiếu niên",
+      "translation": "zone"
+    }
+  ]
```
Comment (kvharish) [inline server/data/words/vietnamese.json:12002]: @NattapolChan Could you please add a new line at the end of the file?

**Comment by drgnz** [comment]
There are lots of wrong translations in `server/data/words/vietnamese.json`

**Comment by jayehernandez** [comment]
Thanks for the comment @drgnz, after this PR is merged would you want to take a look and submit a PR for edits?

**Comment by jayehernandez** [comment]
In line with the comment @drgnz posted, @NattapolChan I'd like to ask if you are a native Vietnamese speaker? I'd really like the translations to be credible for this so native speakers are preferred.

**Pull request closed without merging**